### PR TITLE
feat: streamlined symbiotic veto slasher handling

### PR DESCRIPTION
**Pull request by shaspitz**
## Describe your changes

* Reverts https://github.com/primev/mev-commit/commit/51eed219e5cf8902b570e67c74a5bda85428f60d from https://github.com/primev/mev-commit/pull/457, which addressed https://cantina.xyz/code/4ee8716d-3e0e-4f59-b90d-aa56bf3b484c/findings/563?q=563. The original finding assumed an incorrect behavior of the symbiotic protocol.
* Introduces a more simple approach to handling symbiotic veto slashers in https://github.com/primev/mev-commit/commit/14a4c254ce6eeb7b7052f3975a2c67a322492de1. We simply utilize `vetoDuration` as the min period of time the oracle will be able to call `executeSlashes`, and enforce a 1 hour minimum. More context in changes to README.

## Checklist before requesting a review

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: contracts/contracts/validator-registry/middleware/README.md [hunk 1]
```diff
@@ -108,15 +108,25 @@ For validators who proposed incorrectly as determined by the oracle, slashing mu
 
 ### Instant vs Veto slashers
 
-Vaults with instant slashers must have an `epochDuration` greater than than `slashPeriodSeconds` to register with our middleware contract, ensuring collateral is slashable during the full slashing period. Vaults with veto slashers must have an `epochDuration` greater than `slashPeriodSeconds` + `vetoDuration` + `executeSlashPhaseDuration`, where `vetoDuration` is specified by the slasher. `executeSlashPhaseDuration` is a constant value of 60 minutes for the `MevCommitMiddleware` contract.
+Read more about Symbiotic slashing [here](https://docs.symbiotic.fi/core-modules/vaults#slashing).
 
-Read more about Symbiotic slashing guarantees [here](https://docs.symbiotic.fi/core-modules/vaults#slashing).
+Vaults with instant slashers must have an `epochDuration` greater than than `slashPeriodSeconds` to register with our middleware contract, ensuring collateral is slashable during the full slashing period.
```
Comment (chrmatt) [inline contracts/contracts/validator-registry/middleware/README.md:113]: Typo: "than than"

**Review by chrmatt — COMMENTED** [review]
Looks good to me.

File: contracts/test/validator-registry/middleware/MevCommitMiddlewareTest.sol [hunk 2]
```diff
@@ -73,7 +73,7 @@ contract MevCommitMiddlewareTest is Test {
         vaultFactoryMock = new RegistryMock();
 
         network = vm.addr(0x1);
-        slashPeriodSeconds = 150;
+        slashPeriodSeconds = 150 hours;
```
[the remark this answers is not in the document]
Comment (ckartik) [reply contracts/test/validator-registry/middleware/MevCommitMiddlewareTest.sol:76]: It's a test so doesn't really matter

**Review by Mikelle — APPROVED** [review]

**Review by ckartik — APPROVED** [review]

File: contracts/test/validator-registry/middleware/MevCommitMiddlewareTest.sol (hunk 2 above)
[the remark this answers is not in the document]
Comment (shaspitz) [reply contracts/test/validator-registry/middleware/MevCommitMiddlewareTest.sol:76]: In solidity `hours` still compiles to seconds

**Pull request merged**